src/ui/components/Timeline/index.tsx

```diff
@@ -127,6 +131,24 @@ class Timeline extends Component<PropsFromRedux> {
     }
   };
 
+  isFinishedLoadingRegions() {
+    const { loadingRegions, loadedRegions } = this.props;
+
+    if (
+      !loadingRegions ||
+      !loadedRegions ||
+      loadingRegions.length === 0 ||
+      loadedRegions.length === 0
+    ) {
+      return false;
+    }
+
+    const loading = loadingRegions[0];
+    const loaded = loadedRegions[0];
+
+    return isSameTimeStampedPointRange(loading, loaded);
+  }
```

Comment:
I think this should move to a selector. Also as a cute hack it can bet

```js
    const loading = loadingRegions?.[0];
    const loaded = loadedRegions?.[0];
    return loading && loaded && isSameTimeStampedPointRange(loading, loaded);
```